service/src/main/java/edu/softserve/zoo/service/UserService.java

```diff
@@ -0,0 +1,11 @@
+package edu.softserve.zoo.service;
+
+import edu.softserve.zoo.model.Employee;
+
+/**
+ * @author Ilya Doroshenko
+ */
+public interface UserService {
+    void changePassword(String newPassword, String confirmationPassword);
```

Comment:
confirmation password - it's the old one, maybe not the best naming:)
back-end just sets new password, while, I think, it's front-end's responsibility to ensure that password entered correctly